TZStackView/TZStackView.swift

```diff
@@ -260,11 +270,13 @@ public class TZStackView: UIView {
     public func removeArrangedSubview(view: UIView) {
         if let index = arrangedSubviews.indexOf(view) {
             arrangedSubviews.removeAtIndex(index)
+            view.removeFromSuperview()
```

Comment:
I don't think we should do this, see UIStackView class references:

> \- removeArrangedSubview:
> 
> However, this method does not remove the provided view from the stack’s subviews array; therefore, the view is still displayed as part of the view hierarchy.

And this behavior is explicitly tested by testConsistencyWhenRemovingArrangedSubview method.